game/src/game/player.rs

```diff
@@ -1,17 +1,48 @@
-use theseeker_engine::{gent::{GentPhysicsBundle, TransformGfxFromGent}, animation::SpriteAnimationBundle, script::ScriptPlayer, assets::animation::SpriteAnimation};
+use bevy::{ecs::component::SparseStorage, math::bool, reflect::List};
+use bevy_xpbd_2d::parry::utils::Array1;
+use leafwing_input_manager::{
+    action_state, common_conditions::action_pressed, orientation::Direction, prelude::*,
+};
+use theseeker_engine::{
+    animation::SpriteAnimationBundle,
+    assets::animation::SpriteAnimation,
+    gent::{GentPhysicsBundle, TransformGfxFromGent},
+    script::ScriptPlayer,
+};
 
 use crate::prelude::*;
 
 pub struct PlayerPlugin;
 
 impl Plugin for PlayerPlugin {
     fn build(&self, app: &mut App) {
-        // FIXME: ordering
-        app.add_systems(Update, (setup_player, player_control));
+        // FIXME: ordering, add behavior systems to set and animation systems to set, apply
+        // deffered between
```

Comment:
Don't worry about apply_deferred. Bevy 0.13 is coming out soon and command application points will be managed automatically.